src/main/java/com/kryptnostic/rhizome/hazelcast/serializers/ListStreamSerializers.java

```diff
@@ -14,6 +16,39 @@
  * @author Matthew Tamayo-Rios &lt;matthew@kryptnostic.com&gt;
  */
 public class ListStreamSerializers {
+
+    public static <T> void serialize(
```

Comment:
Did you consider using `ListStreamSerializer`?

Reply:
That class only has one method in it

Reply:
Agreed that this is preferable to ListStreamSerializer

Reply:
@pckeyes By using that I meant to extend it instead of creating static methods.
@makosblade Can you pls elaborate why?

Reply:
I just think for serializing a list or a collection in a generic way, creating a type hierarchy (abstract class `ListStreamSerializer` extends `CollectionStreamSerializer`, etc) just adds complexity to what is otherwise a pretty simple goal: take a class, turn it into bytes efficiently. 

I think that goal is better accomplished by having static methods that we can re-use for any list, or slightly more efficient methods for specific kinds of lists (serializing `List<UUID>` as one or two `long[]` instead of `String[]` for example)

Reply:
and in fact, we can replace all generic `List`, `Collection`, `Set` default serializers with a generic `Iterable` serializer and just be done with it 😆 
Of course it would be nice to keep around some extra-efficient List/Set/Collection serializers for specific cases

Reply:
In this case I specifically needed a static method to be used in the OptionalStreamSerializers class